### PR TITLE
Feature/list view

### PR DESCRIPTION
**Pull request by deonjo**
Hey guys,

Big feature. I added a way to view our data in a list instead of just in groups. This is going to become the major way of data input. Such as inputing class/age, gender, location. 

Added a unit test to make sure that it renders properly. In the snapshot, it renders as just a div, but that's because Ag-Grid hasn't fully rendered yet. But it goes through all the internal logic of creating all the parts that will get rendered.

Here's a screenshot of the feature below.
Screenshot 2019-06-03 20 28 16

### CODE REVIEW

File: src/components/editContainer/__snapshots__/editContainer.unit.test.js.snap [hunk 1]
```diff
@@ -59,63 +59,3 @@ exports[`EditContainer renders 1`] = `
   </div>
 </div>
 `;
-
-exports[`NavAppBar is consistent 1`] = `
```
Comment (gabekoo) [inline src/components/editContainer/__snapshots__/editContainer.unit.test.js.snap]: Double check that the snapshot is correct

File: src/containers/listView/containerCellEditor.jsx [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+import MenuItem from '@material-ui/core/MenuItem';
+
+export class ContainerCellEditor extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            containerId: (this.props.value) ? this.props.value._id : "",
+            open: false
+        }
+        // this.props.value is the currently selected value
```
Comment (gabekoo) [inline src/containers/listView/containerCellEditor.jsx]: Do you want to rename this.props.value to something like this.props.currentValue to make it more clear, based on this comment?

Comment (deonjo) [reply src/containers/listView/containerCellEditor.jsx]: I can't name them. They are named by the third party component Ag-Grid. This is just a note for myself.

File: src/components/editContainer/__snapshots__/editContainer.unit.test.js.snap (hunk 1 above)
Comment (deonjo) [reply src/components/editContainer/__snapshots__/editContainer.unit.test.js.snap]: This is actually correct. Good catch. At a point in time, the test was renamed, but the snapshot wasn't updated. I should add a precommit hook to run the tests.

**Pull request merged**